Calculator_Project/Source.cpp

```diff
@@ -0,0 +1,251 @@
+                equ_vec.push_back(num_str);
+            }
+            else if (char_is_num(equ_str[equ_str_index + 1]) == false) {
+                equ_vec.push_back(num_str);
+            }
+            break;
+        case '6':
+            if (equ_str_index == 0) {
+                num_str = "6";
+            }
+            else if (char_is_num(equ_str[equ_str_index - 1]) == false) {
+                num_str = "6";
+            }
+            else {
+                num_str.append("6");
+            }
+
+            if (equ_str_index == (equ_str.length() - 1)) {
+                equ_vec.push_back(num_str);
+            }
+            else if (char_is_num(equ_str[equ_str_index + 1]) == false) {
+                equ_vec.push_back(num_str);
+            }
+            break;
+        case '7':
+            if (equ_str_index == 0) {
+                num_str = "7";
+            }
+            else if (char_is_num(equ_str[equ_str_index - 1]) == false) {
+                num_str = "7";
+            }
+            else {
+                num_str.append("7");
+            }
+
+            if (equ_str_index == (equ_str.length() - 1)) {
+                equ_vec.push_back(num_str);
+            }
+            else if (char_is_num(equ_str[equ_str_index + 1]) == false) {
+                equ_vec.push_back(num_str);
+            }
+            break;
+        case '8':
+            if (equ_str_index == 0) {
+                num_str = "8";
+            }
+            else if (char_is_num(equ_str[equ_str_index - 1]) == false) {
+                num_str = "8";
+            }
+            else {
+                num_str.append("8");
+            }
+
+            if (equ_str_index == (equ_str.length() - 1)) {
+                equ_vec.push_back(num_str);
+            }
+            else if (char_is_num(equ_str[equ_str_index + 1]) == false) {
+                equ_vec.push_back(num_str);
+            }
+            break;
+        case '9':
+            if (equ_str_index == 0) {
+                num_str = "9";
+            }
+            else if (char_is_num(equ_str[equ_str_index - 1]) == false) {
+                num_str = "9";
+            }
+            else {
+                num_str.append("9");
+            }
+
+            if (equ_str_index == (equ_str.length() - 1)) {
+                equ_vec.push_back(num_str);
+            }
+            else if (char_is_num(equ_str[equ_str_index + 1]) == false) {
+                equ_vec.push_back(num_str);
+            }
+            break;
+        case '+':                                       //If the current character is a +,
+            equ_vec.push_back("+");                     //append it directly to the equation vector
+            break;
+        default:                                        //If the current character is anything else,
+            std::cout << "ERROR: Undefined character!"; //print error message - needs actual error handling process?

```

Comment:
Probably should stop parsing the string entirely at this point, because it's likely impossible to still evaluate the string if the input is messed up here